src/Search_Replace_Command.php

```diff
@@ -165,10 +168,14 @@ public function __invoke( $args, $assoc_args ) {
 		$total           = 0;
 		$report          = array();
 		$this->dry_run         = \WP_CLI\Utils\get_flag_value( $assoc_args, 'dry-run' );
-		$php_only        = \WP_CLI\Utils\get_flag_value( $assoc_args, 'precise' );
+		$php_only              = \WP_CLI\Utils\get_flag_value( $assoc_args, 'precise' );
 		$this->recurse_objects = \WP_CLI\Utils\get_flag_value( $assoc_args, 'recurse-objects', true );
 		$this->verbose         =  \WP_CLI\Utils\get_flag_value( $assoc_args, 'verbose' );
 		$this->format          = \WP_CLI\Utils\get_flag_value( $assoc_args, 'format' );
+		$this->regex_limit    = \WP_CLI\Utils\get_flag_value( $assoc_args, 'regex-limit', -1 );
```

Comment:
To follow what's there now this should be within the `if ( ( $this->regex = ...` block below.

(I note we're not currently checking if `--regex-` options used when `--regex` not given which we should do but that's a separate PR.)